incremental_alter_configs_response.go

```diff
@@ -0,0 +1,66 @@
+package sarama
+
+import "time"
+
+// IncrementalAlterConfigsResponse is a response type for incremental alter config
+type IncrementalAlterConfigsResponse struct {
+	ThrottleTime time.Duration
+	Resources    []*AlterConfigsResourceResponse
```

Comment:
In this place I have used `AlterConfigsResourceResponse` - thats why file with according definition is also affected. In same way it is used in Kafka Java Client.